groups/infrastructure/module-secrets/parameters.tf

```diff
@@ -0,0 +1,9 @@
+resource "aws_ssm_parameter" "secret_parameters" {
+  for_each = var.secrets
+    name  = "/${var.name_prefix}/${each.key}"
+    key_id = var.kms_key_id
+    description = each.key
+    type  = "SecureString"
+    overwrite = "true"
+    value = each.value
+}
```

Comment:
Can you please address the indentation and parameter alignment?
E.g.:
```
resource "aws_ssm_parameter" "secret_parameters" {
  for_each = var.secrets
  
  name        = "/${var.name_prefix}/${each.key}"
  key_id      = var.kms_key_id
  description = each.key
  ```
  
  and so on